scripts/feature_tests/upgrade/upgrade.sh

```diff
@@ -23,6 +23,10 @@ source "${METAL3_DEV_ENV_DIR}/lib/network.sh"
 # shellcheck disable=SC1090
 source "${METAL3_DEV_ENV_DIR}/lib/images.sh"
 
+# shellcheck disable=SC1091
+# shellcheck disable=SC1090
+source "${METAL3_DEV_ENV_DIR}/vm-setup/roles/v1aX_integration_test/files/manifests/clusterctl_env_${IMAGE_OS}.rc"
+
```

Comment:
what variables are you using from this file ? I am a bit concerned to rely on a file that was generated for a temporary use in ansible, and might be modified there, without realizing that it would impact this case, and breaking it. If you do not use many variables it might be better to define those variables here, or re-generate the file

Reply:
for the cluster provisioning:
API_ENDPOINT_HOST: 192.168.111.249
API_ENDPOINT_PORT: 6443
POD_CIDR: 192.168.0.0/18
SERVICE_CIDR: 10.96.0.0/12
and several other for control plane and worker provisioning

Reply:
removed